fili-core/src/test/groovy/com/yahoo/bard/webservice/data/DruidHavingBuilderSpec.groovy

```diff
@@ -24,7 +24,7 @@ public class DruidHavingBuilderSpec extends Specification {
             .registerModule(new Jdk8Module().configureAbsentsAsNulls(false))
 
     def setupSpec() {
-        resources = new QueryBuildingTestingResources()
+        resources = new QueryBuildingTestingResources().init()
```

Comment:
`QueryBuildingTestingResources` now extends `spock.Specification` which does not allow constructor, so we have to put all initialization in a `init()` method.